app/models/discussion.rb

```diff
@@ -193,6 +193,10 @@ def inherit_group_privacy!
     end
   end
 
+  def example_discussion?
+    self.author_id == User.loomio_helper_bot.id
```

Comment:
This pretty much sucks, and I think we probably want to refactor and add an 'example' column to Discussion so we don't have to bother the poor helper bot 1000s of times for each discussion we load.